rigex-front-end/public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="%PUBLIC_URL%/assets/fonts.css" />
```

Comment:
Could you clarify which? The line you highlight is a font import

Reply:
Yeah it doesn't let me comment on the .png files. Specifically referencing the mail-icon.png Since we get a mail icon from material ui I don't think we need that asset

Reply:
ahh I gotchya - fixed